docs/modules/ROOT/pages/performance.adoc

```diff
@@ -585,14 +585,17 @@ link:https://wiki.mozilla.org/Security/Server_Side_TLS[Mozilla Server Side TLS])
 [source, java]
 ----
 
-// add the netty dependency to your build, eg: "io.netty:netty-tcnative-boringssl-static:2.0.25.Final"
+// add the netty dependency to your build, eg: "io.netty:netty-tcnative-boringssl-static:$tcnativeVersion"
 
 BlockingHttpClient client = HttpClients.forSingleAddress("servicetalk.io", 443)
         .secure().provider(SecurityConfigurator.SslProvider.OPENSSL).commit()
         .buildBlocking();
 HttpResponse resp = client.request(client.get("/"));
 ----
 
+Latest `netty-tcnative-boringssl-static` version can be found
+link:https://mvnrepository.com/artifact/io.netty/netty-tcnative-boringssl-static[here].
```

Comment:
Perhaps [maven.org](https://search.maven.org/artifact/io.netty/netty-tcnative-boringssl-static) instead since the artifact is hosted on maven central?

Web accessibility discourages "here" links so perhaps either "maven.org" or "mvnrepository" as the link title?